src/TableGateway/TableGatewayInterface.php

```diff
@@ -9,11 +9,47 @@
 
 namespace Zend\Db\TableGateway;
 
+use Zend\Db\ResultSet\ResultSetInterface;
+use Zend\Db\Sql\Where;
+
 interface TableGatewayInterface
 {
+    /**
+     * @return string
+     */
     public function getTable();
+
+    /**
+     * Select
+     *
+     * @param  Where|\Closure|string|array $where
+     * @return ResultSetInterface
+     */
     public function select($where = null);
+
+    /**
+     * Insert
+     *
+     * @param  array $set
+     * @return int
+     */
     public function insert($set);
+
+    /**
+     * Update
+     *
+     * @param  array $set
+     * @param  string|array|\Closure $where
+     *
+     * @return int
+     */
     public function update($set, $where = null);
+
+    /**
+     * Delete
```

Comment:
Overall, these one-word descriptions are not useful. Drop them overall unless there is something to clarify further